keyboards/treasure/type9s2/readme.md

```diff
@@ -0,0 +1,15 @@
+# Treasure TYPE-9 Series II
+
+![TYPE9SII](https://i.imgur.com/lohJ0Da.png)
+
+Treasure TYPE-9 Series II is a 3x3 Macropad.
+
+* Keyboard Maintainer: [Treasure](https://github.com/TreasureTypes)
+* Hardware Supported: Treasure TYPE-9 Series II PCB
+* Hardware Availability: [Treasure Store](http://treasuretypes.com)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader?  So that new firmware can be flashed.

Reply:
the instructions has been added.